napi/transform/index.d.ts

```diff
@@ -203,6 +203,21 @@ export interface TransformOptions {
   typescript?: TypeScriptOptions
   /** Configure how TSX and JSX are transformed. */
   jsx?: JsxOptions
+  /**
+   * Sets the target environment for the generated JavaScript.
+   *
+   * The lowest target is `es2015`.
+   *
+   * Example:
+   *
+   * * 'es2015'
+   * * ['es2020', 'chrome58', 'edge16', 'firefox57', 'node12', 'safari11']
+   *
+   * @default `esnext` (No transformation)
+   *
+   * @see [esbuild#target](<https://esbuild.github.io/api/#target)
+   */
+  target?: string | Array<string>
```

Comment:
Can we make it type-safe? For example: 
```ts
{
 target: 'es2015' | 'es2016' | Array<`chrome${string}` | `edge${string}`>
}
```

Reply:
It would be

```
type Target = 'es2015' | ... 
Target | Array<Target>
```

I'll do this later when I get time.